### DISCUSSION TITLE
Support mat-highlight-text (or equivalent of md-highlight-text)

### DISCUSSION THREAD

**Discussion by florin05** [angular/components#25190] [Ideas]
#### Bug, feature request, or proposal: 
Feature request

#### What is the expected behavior? 
Support an equivalent of md-highlight-text

#### What is the current behavior? d
No support text highlighting directive

#### Which versions of Angular, Material, OS, TypeScript, browsers are affected? 
5

#1042 is the same issue, was linked with autocomplete and closed when that was released, but highlighting feature is not there, so opening this.

[a comment by an automated account is not shown]

**Comment by amysorto** [comment]
For context, AngularJS had a directive called `md-highlight-text` that was applied to some text with specified text to highlight.  For example `<p md-highlight-text=“car”>racecar</p>` would appear visually as race**car** (the word car would be highlighted in this case).

This directive was used in the AngularJS autocomplete component, where the suggested options would have this highlight styling on the matching letters. 

Should this directive still exist and/or should it be a part of the autocomplete component?

**Comment by OhanyanSofia** [comment]
?
